src/main/java/core/basesyntax/service/strategy/SubtractOperationHandler.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.service.strategy;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.DailyActivity;
+import core.basesyntax.model.Fruit;
+
+public class SubtractOperationHandler implements OperationHandler {
+    @Override
+    public boolean operation(DailyActivity dailyActivity) {
```

Comment:
if you always return true by default, you can make the method return void to return false instead of throwing an exception

Reply:
but, l thought it`s much better to create methods with return type(at least boolean) to avoid troubles during testing of this method